pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -75,6 +70,15 @@
 )
 from pytorch_lightning.utilities.enums import PrecisionType
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
+from pytorch_lightning.utilities.imports import (
+    _APEX_AVAILABLE,
+    _HOROVOD_AVAILABLE,
+    _IPU_AVAILABLE,
+    _NATIVE_AMP_AVAILABLE,
+    _TORCH_GREATER_EQUAL_1_7,
+    _TORCH_GREATER_EQUAL_1_8,
+    _TPU_AVAILABLE,
+)
```

Comment:
Is separating these imports for readability or sth else?

Reply:
for readability and to help me figure out which exact module exports which symbols. it can be hard to tell when everything comes from `pytorch_lightning.utilities`